lib/Avatar/index.js

```diff
@@ -5,66 +5,55 @@ import cx from 'classnames';
 /**
  * @usage
  *
- * <Avatar initials="MR" color="green" url="img/url" onlyInitials isHighlighted offline />
+ * <Avatar name="Seymour Butts" color="green" url="img/url" offline />
  */
-const Avatar = props => {
-  const offlineBg = Avatar.defaultProps.colour;
-  const displayImage = props.url && !props.onlyInitials && !props.offline;
-  const displayGreyedImage = props.url && props.offline;
-  const backgroundColor = props.offline ? offlineBg : props.colour;
+const getInitialsFromName = name => {
+  const names = name.split(' ');
+  return names.reduce((acc, value) => `${acc}${value.split('')[0]}`, '');
```

Comment:
You can just do `value[0]` instead of `value.split('')[[0]` here, to get the first character from a string.

Also I think we pass the initials with the API anyway, and the logic Angus decided upon was we would just take the first letter from the first and last names, so we should always have two characters. What happens with the above reducer if the person has multiple names, represented by spaces? You could potentially get three or more characters.